lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -40,12 +40,26 @@
 
 
 class CalibrationResults:
-    """Power spectrum calibration results."""
+    """Power spectrum calibration results.
 
-    def __init__(self, model, ps_model_fit, ps_fitted, params, results):
+    Attributes
+    ----------
+    model : `lumicks.pylake.force_calibration.CalibrationModel`
+        Model used for calibration.
+    ps_fitted_model : `lumicks.pylake.PowerSpectrum`
+        Power spectrum of the fitted model.
+    ps_fitted_data : `lumicks.pylake.PowerSpectrum`
+        Power spectrum of the data that the model was fitted to.
```

Comment:
I think that simply `ps_model` and `ps_data` would be pretty unambiguous. Any reason to specifically include `fitted`? Do we ever have anything that's not fitted?

Reply:
I guess not 😅 
I'll change it :)